tests/integration_tests/build/test_binary_szie.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Test that checks if the release binary size is within expected size"""
+
+import os
+import pytest
+
+import host_tools.cargo_build as host
+
+BINARY_SIZE_TARGET = 3137280
+"""Target binary size to be checked in bytes"""
+
+BINARY_SIZE_DELTA = 1024 * 1024 / 2
```

Comment:
I would use a percentage here. Also 0.5MB is a lot, we do not usually have PRs that are that beefy in terms of binary output, unless we add extra dependencies which we should also avoid.

Reply:
Done